tendrl/node_agent/manager/tendrl_definitions_node_agent.py

```diff
@@ -137,6 +137,16 @@
           run: tendrl.node_agent.atoms.service.configure.Configure
           type: Update
           uuid: b90a0d97-8c9f-4ab1-8f64-dbb5638159a3
+        check_service_status:
+          enabled: true
+          inputs:
+            mandatory:
+              - Node.fqdn
+              - Service.name
+          name: "check whether the service is running"
+          run: tendrl.node_agent.objects.service.atoms.check_service_status.CheckServiceStatus
+          type: Create
```

Comment:
I felt that while writing this but I found this used for check_node_up which is a similar thing(basically a check atom)